SophieUdud/JobTracking
# Resume report UI

Steps to test:

1. Add a new resource in your scratchorg called ChartJs, and upload the Chart.bundle.min.js

2. Create several applications

3. Upload a few resumes to a few applications

4. Attach the same resumes to the other applications (make sure you don't re-upload from disk the same resume name again, it will actually be a different Salesforce object)

5, Go to the application page, and edit page, search for barchart and add the parentbarchart to the UI

![image](https://user-images.githubusercontent.com/5331315/146459528-28652241-4853-4f19-b261-3726c038b2cb.png)
